src/pubsub.rs

```diff
@@ -239,23 +239,22 @@ impl PubSub {
     /// Enqueue a message to be sent to the broker under the given topic
     pub fn enqueue_bytes<S>(&mut self, topic: S, buf: &[u8])
     where
-        S: Into<String>,
+        S: AsRef<str>,
```

Comment:
Good change, did this have a meaningful perf improvement?

Reply:
yeah, h/t josiah